arches_lingo/src/arches_lingo/components/scheme/report/SchemeStandard.vue

```diff
@@ -62,25 +65,36 @@ async function getOptions(): Promise<ResourceInstanceReference[]> {
 
 async function save() {
     try {
-        await updateSchemeCreation(
-            route.params.id as string,
-            schemeInstance.value as SchemeInstance,
-        );
+        let updated;
+        if (route.params.id === NEW) {
+            updated = await createScheme(schemeInstance.value);
+            router.push({
+                name: "scheme",
+                params: { id: updated.resourceinstanceid },
+            });
+        } else {
+            updated = await updateSchemeCreation(
+                route.params.id as string,
+                schemeInstance.value,
+            );
+        }
+        schemeInstance.value = updated;
+        emit(UPDATED);
     } catch (error) {
         toast.add({
             severity: ERROR,
-            summary: $gettext("Error"),
-            detail:
-                error instanceof Error
-                    ? error.message
-                    : $gettext("Could not save the scheme standard"),
+            summary: $gettext("Error saving scheme"),
+            detail: (error as Error).message,
```

Comment:
I can back out this change if you don't like it, I was just trying to minimize the number of translated strings ('standard' should really be a param we can feed)